skbeam/core/accumulators/binned_statistic.py

```diff
@@ -79,9 +79,10 @@ def __init__(self, sample, statistic='mean',
         from 1.10.0 onwards.
         """
 
-        known_stats = ['mean', 'median', 'count', 'sum', 'std']
+        self.known_stats = ['mean', 'median', 'count', 'sum', 'std']
         self.statistic = statistic
-        if not callable(self.statistic) and self.statistic not in known_stats:
+        if not callable(
+                self.statistic) and self.statistic not in self.known_stats:
```

Comment:
Since you duplicate this validation logic twice, consider doing it as a setter function instead